src/templates/Landing.js

```diff
@@ -239,7 +244,7 @@ export default props => {
             />
           );
         }, nodes),
-        <ViewAllCard {...viewAllProps} />,
+        <ViewAllCard {...viewAllProps} {...{className}} />,
```

Comment:
Or this if its supposed to be `classNames`

Reply:
It would, if it were being combined with other classNames. In this case it's not